priv/templates/partials/head.html.eex

```diff
@@ -3,6 +3,12 @@
   <meta charset="utf-8">
   <meta http-equiv="content-type" content="text/html; charset=utf8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" type="text/css" href="assets/stylesheets/benchee.css">
-  <link rel="stylesheet" type="text/css" href="assets/fontello/css/fontello.css">
+  <%= if inline_assets do %>
+    <style>
+      <%= File.read! Path.join(Application.app_dir(:benchee_html, "priv/assets/stylesheets/"), "benchee.css") %>
```

Comment:
seems like the `fontello` css is omitted? Or is that because there was some sort of problem there?

Reply:
The reason I omitted fonts is because I don't know how to embed them in html easily. I can change fontello to use svg font and then inline this into html or we can use default font in inllined version. What do you think?

Reply:
didn't notice they were separate paths :see_no_evil: 

All good. I think the problem is that the help icon doesn't show up then but maybe we can treat that some other way. I think that was about the only reason to add the font either way 2999ffbf589a09d

But I think we can handle that in a separate issue/PR - if that's really the only reason we use it, we might be able to get away with something simpler than a whole font for it.